src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -0,0 +1,375 @@
+@import '../Media/Media.scss';
+@import '../Variables.scss';
+
+$tablet-small-end: 670px;
+
+.flex {
+  display: flex;
+}
+
+.itemsCenter {
+  align-items: center;
+}
+
+.hamburger {
+  position: fixed;
+  top: 24px;
+  right: 24px;
+  z-index: 2;
+
+  @include for-laptop-and-up {
+    display: none;
+  }
+}
+
+.phoneAndTablet {
+  position: relative;
+  height: var(--Space-64);
+  background-color: white;
+  padding: var(--Space-24) var(--ContainerPadding--Phone);
+  display: none;
+  align-items: center;
+  justify-content: flex-end;
+  display: flex;
+  @include for-laptop-and-up {
+    display: none;
+  }
+  &.visible {
+    z-index: 3;
+  }
+  .hamburger {
+    position: absolute;
+  }
+}
+
+.laptopAndUp {
+  height: var(--Space-64);
+  background-color: white;
```

Comment:
`--White`

Reply:
hmm, thought i had vars in use here, will update